### PR TITLE
Added broken dark mode toggle

### PR DESCRIPTION
**Pull request by Lane-G-Smith**
work in progress
need to fix a state change issue
translating to react from a non-react project

### CODE REVIEW

**Review by akpi816218 — APPROVED** [review]

**Comment by akpi816218** [comment]
Will type these within a couple days of merging, time may vary based on the amount needed.

**Comment by Lane-G-Smith** [comment]
I'll have to come back to this. I need the dark mode button to work with useState and I'm not yet sure how to do that.

File: src/components/DarkMode/DarkMode.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// DARK MODE
+
+// attach event listener to theme toggle button / icon
+const themeToggle = document.getElementById('theme-toggle')!;
+themeToggle.addEventListener('click', () => {
```
Comment (akpi816218) [inline src/components/DarkMode/DarkMode.ts:5]: Not working... Throws every time

**Comment by akpi816218** [comment]
> I'll have to come back to this. I need the dark mode button to work with useState and I'm not yet sure how to do that.

@Lane-G-Smith `document.getElementById()` is not returning a `HTMLElement`. It works in the console... I think it's running before the stuff is rendered. But you probably know more about this than me.

**Review by akpi816218 — APPROVED** [review]
Merging basic changes is ok for now, need to fix button toggle

**Pull request merged**